content/powershell/concepts/operators/operators.md

```diff
@@ -1,14 +1,122 @@
+|   `-`    |  Subtraction   | Subtracts or negates numbers                                                                |
+|   `*`    | Multiplication | Multiplies numbers or copies strings and arrays a specified number of times                 |
+|   `/`    |    Division    | Divides numbers                                                                             |
+|   `%`    |     Modulo     | Returns the remainder of a division operation                                               |
+
+Arithmetic operators are binary operators. Their syntax in PowerShell is `<Operand_1> <Arithmetic Operator> <Operand_2>`.
+
+```PowerShell
+$x = 5 + 5  # x is now 10
+$x = $x - 8 # x is now 2
+$x = $x * 3 # x is now 6
+$x = $x / 2 # x is now 3
+$x = $x % 2 # x is now 1
+```
+
+Arithmetic operators, `+` and `*`, also work on strings and arrays.
+```PowerShell
+PS > $best_learning_platform = "code" + "cademy"
+PS > $best_learning_platform + "!" * 3
+codecademy!!!
+```
+
+```PowerShell
+PS > $fibonacci_1 = 0, 1, 1
+PS > $fibonacci_2 = 2, 3, 5
+PS > $fibonacci_1 + $fibonacci_2
+0
+1
+1
+2
+3
+5
+
+PS > $fibonacci_2 * 2
+2
+3
+5
+2
+3
+5
+```
+
+## Assignment Operators
+Assignment operators can be used to assign, change, or append values to variables. These operators are a shorter syntax for assigning the result of an arithmetic operator. The general syntax of the assignment operators is: `<Variable> <Assignment Operator> <Value>`.
+
+| Operator |                Name                | Description                                 |
+| :------: | :--------------------------------: | ------------------------------------------- |
+|   `=`    |             Assignment             | `$x = 3` assigns value `3` to variable `x`. |
+|   `+=`   |    Addition Compound Assignment    | `$x += 3` is short for `$x = $x + 3`.       |
+|   `-=`   |  Subtraction Compound Assignment   | `$x -= 3` is short for `$x = $x - 3`.       |
+|   `*=`   | Multiplication Compound Assignment | `$x *= 3` is short for `$x = $x * 3`.       |
+|   `/=`   |    Division Compound Assignment    | `$x /= 3` is short for `$x = $x / 3`.       |
+|   `%=`   |     Modulo Compound Assignment     | `$x %= 3` is short for `$x = $x % 3`.       |
+
+## Unary Operators
+Unary operators increase or decrease the value of a variable by 1.
+
+| Operator |      Name      | Description                                                   |
+| :------: | :------------: | ------------------------------------------------------------- |
+|   `++`   |   Increment    | `$x++` returns `$x + 1`                                       |
+|   `--`   |   Decrement    | `$x--` returns `$x - 1`                                       |
+
+## Equality Comparison Operators
+Equality operators in PowerShell are binary operators that compare two integer or string values that return `True` if the operator condition is met, otherwise `False`.
```

Comment:
```suggestion

Equality operators in PowerShell are binary operators that compare two integer or string values that return `True` if the operator condition is met, otherwise `False`.
```